.github/actions/setup-arc-e2e/action.yaml

```diff
@@ -20,30 +20,30 @@ inputs:
 
 outputs:
   token:
-    description: 'Token to use for configure ARC'
+    description: "Token to use for configure ARC"
     value: ${{steps.config-token.outputs.token}}
 
 runs:
   using: "composite"
   steps:
     - name: Set up Docker Buildx
-      uses: docker/setup-buildx-action@v3
+      uses: docker/setup-buildx-action@b5ca514318bd6ebac0fb2aedd5d36ec1b5c232a2
```

Comment:
Thinking out loud, it's going to be nightmare upgrading these later if we don't know to which version they point to exactly.

How do you feel about adding a comment above each one to reference the human readable tag that the SHA points to?